src/rules/exponential/out.jl

```diff
@@ -0,0 +1,6 @@
+export rule
+using Distributions
+
+@rule Exponential(:out, Marginalisation) (m_in1::PointMass, ) = begin
+    return PointMass(exp(mean(m_in1)))
```

Comment:
I don't think this is correct. It should be
```
return Exponential(mean(m_in1))
```

But I can see indeed that in `ForneyLab` its just `exp`. What do you think @bartvanerp? It seems to me that in ForneyLab `Exponential` node is not a distribution, but a deterministic `exp` function. It has the same name but the meaning is not quite the same, right?

EDIT: My assumption is correct, in ForneyLab its just an `exp` mapping, see [here](https://github.com/biaslab/ForneyLab.jl/blob/master/src/factor_nodes/exponential.jl). We will think about it with @bartvanerp. You can skip this one for now @A-Vzer 

Also better to avoid uninformative names in actual distributions, for [Exponential distribution](https://en.wikipedia.org/wiki/Exponential_distribution) instead of `in1` its better to use `\lambda<TAB>`